invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/License/LicenseField.js

```diff
@@ -73,9 +73,21 @@ class LicenseFieldForm extends Component {
 
     const uiRights = getIn(values, uiFieldPath, []);
 
+    const licenseList = getIn(values, fieldPath, []);
+    const formikInitialValues = getIn(initialValues, fieldPath, []);
+
+    const error = getIn(errors, fieldPath, null);
+    const initialError = getIn(initialErrors, fieldPath, null);
+    const licenseError = error || (licenseList === formikInitialValues && initialError);
+
+    let className = "";
+    if (licenseError) {
+      className = typeof licenseError !== "string" ? licenseError.severity : "error";
+    }
```

Comment:
see my previous comment on contributors:
```suggestion
    let className = license.severity;
```